docs/geps/gep-05.rst

```diff
@@ -0,0 +1,198 @@
+.. _gep-5:
+
+=============================================
+GEP 5 — Optional Rounding of Variables
+=============================================
+
++------------+------------------------------------------------------------------+
+| Author     | `Christian Zimpelmann <https://github.com/ChristianZimpelmann>`_ |
++            +------------------------------------------------------------------+
+|            | `Janos Gabler <https://github.com/janosg>`_                      |
```

Comment:
@hmgaudecker, would you also like to be added?

Reply:
No, I just added annoying comments, few constructive stuff!